src/axios-services/index.js

```diff
@@ -20,14 +20,10 @@ import axios from 'axios';
 
 export async function getAPIHealth() {
   try {
-    const { data } = await axios.get('/api/health');
+    const { data } = await axios.get("https://localhost:4000/api/health");
```

Comment:
Don't think this should change. Should keep it as '/api/health'

Reply:
Sure that sounds good. I only wanted to switch it because in my browser it keeps loading to localhost:3000. I just have been changing it manually.

Reply:
That's intentional. The front-end is using localhost:3000 & the api is using localhost:4000.

Reply:
You are right. Since I made the switch to WSL, the page is finally rendering correctly.